README.md

```diff
@@ -79,6 +79,15 @@ Use `platypus.jump()` to perform a jump and `platypus.abort_jump()` to reduce th
 			end
 		end
 	end
+    
+## Double jump, wall jump and wall slide
+Platypus supports double jumps when config.allow_double_jump is set to true. Then, it occurs automatically, when used jump() while already jumped up.
```

Comment:
Change second sentence to "A double jump is performed automatically when a second jump is done before and up until reaching the apex of the first jump. It is not possible to performa a double jump when falling"